### PR TITLE
Translate traffic-shifting task to Chinese

### PR DESCRIPTION
**Pull request by zhaohuabing**
Signed-off-by: Huabing Zhao <zhaohuabing@gmail.com>

### CODE REVIEW

**Comment by geeknoid** [comment]
Please update this PR to use the new annotations for code blocks. Instead of backticks, you need to use

{{< text bash >}}

Please see [here](https://preliminary.istio.io/about/contribute/writing-a-new-topic/#embedding-preformatted-blocks) for info

Thanks.

**Comment by zhaohuabing** [comment]
@gyliu513 @geeknoid Could we merge this PR now? The ci failure is not relevant to this PR.

```
- ./public/docs/tasks/telemetry/fluentd/index.html
  *  213:53: ERROR: Unexpected end tag : p (line 213)
htmlproofer 3.9.1 | Error:  HTML-Proofer found 1 failure!
```

File: content_zh/docs/tasks/traffic-management/traffic-shifting/index.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+1.  首先，使用下面的命令把50%的流量从 `reviews:v1` 转移到 `reviews:v3`:
+
+    {{< text bash >}}
+    $ istioctl replace -f @samples/bookinfo/routing/route-rule-reviews-50-v3.yaml@
+    {{< /text >}}
+
+    确认规则已被替换:
+
+    {{< text yaml >}}
+        $ istioctl get virtualservice reviews -o yaml
```
Comment (geeknoid) [inline content_zh/docs/tasks/traffic-management/traffic-shifting/index.md]: Remove indent, the content should line up with the {{< text yaml >}}

**Comment by geeknoid** [comment]
Could you please update your branch with the latest changes and resubmit? From the analysis failure, I can see this is based on an older copy of the repo.

Thanks.

**Comment by zhaohuabing** [comment]
I'm modifying the translation according to the latest update of the English version:https://github.com/istio/istio.github.io/commit/d5a724dfd67f0e40b0c4565cf0e5e5ff060216fb
Please hold on.

File: content_zh/docs/tasks/traffic-management/traffic-shifting/index.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+一个常见的用例是将流量从一个版本的微服务逐渐迁移到另一个版本。 在Istio中，您可以通过配置一系列规则来实现此目标，
+这些规则将一定百分比的流量路由到一个或另一个服务。 在此任务中，您将先分别向 `reviews:v1` 和 `reviews:v3` 各发送50%流量。
+然后，您将通过向 `reviews:v3` 发送100％的流量来完成迁移。
+
+## 应用基于权重的路由
+
+1.  首先，运行此命令将所有流量路由到 `v1` 版本的各个微服务。
+
+    {{< text bash >}}
+    $ istioctl create -f @samples/bookinfo/routing/route-rule-all-v1.yaml@
```
Comment (fleeto) [inline content_zh/docs/tasks/traffic-management/traffic-shifting/index.md]: 更新成 @networking/virtual-service-all-v1.yaml@ ？
目前的文件引用基于 release-1.0 的相对位置。

File: content_zh/docs/tasks/traffic-management/traffic-shifting/index.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+1.  在浏览器中打开 Bookinfo 站点。 URL为 `http：// $ GATEWAY_URL / productpage`，其中 `$GATEWAY_URL`是 ingress 的外部IP地址，
+其描述参见 [Bookinfo](/docs/examples/bookinfo/#determining-the-ingress-ip-and-port)。
+
+     请注意，不管刷新多少次，页面的评论部分都不会显示评级星号。这是因为 Istio 被配置为将 reviews 服务的的所有流量都路由到了 `reviews：v1` 版本，
+     而该版本的服务不会访问带星级的 ratings 服务。
+
+1.  使用下面的命令把50%的流量从 `reviews:v1` 转移到 `reviews:v3`:
+
+    {{< text bash >}}
+    $ istioctl replace -f @samples/bookinfo/routing/route-rule-reviews-50-v3.yaml@
```
Comment (fleeto) [inline content_zh/docs/tasks/traffic-management/traffic-shifting/index.md]: virtual-service-reviews-50-v3.yaml

File: content_zh/docs/tasks/traffic-management/traffic-shifting/index.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    {{< /text >}}
+
+1.  刷新浏览器中的 `/productpage` 页面，大约有50%的几率会看到页面中出带红色星级的评价内容。这是因为 `v3` 版本的 `reviews` 访问了带星级评级的 `ratings` 服务，但`v1`版本却没有。
+
+    > 在目前的Envoy sidecar实现中，可能需要刷新 `/productpage` 很多次--可能15次或更多--才能看到流量分发的效果。您可以通过修改规则将90%的流量路由到v3，这样看到更多带红色星级的评价。
+
+1. 如果您认为 `reviews：v3` 微服务已经稳定，你可以通过应用此 virtual service 将100％的流量路由到 `reviews：v3`：
+
+    {{< text bash >}}
+    $ istioctl replace -f @samples/bookinfo/routing/route-rule-reviews-v3.yaml@
```
Comment (fleeto) [inline content_zh/docs/tasks/traffic-management/traffic-shifting/index.md]: virtual-service-reviews-v3.yaml

File: content_zh/docs/tasks/traffic-management/traffic-shifting/index.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+## 应用基于权重的路由
+
+1.  首先，运行此命令将所有流量路由到 `v1` 版本的各个微服务。
+
+    {{< text bash >}}
+    $ istioctl create -f @samples/bookinfo/routing/route-rule-all-v1.yaml@
+    {{< /text >}}
+
+1.  在浏览器中打开 Bookinfo 站点。 URL为 `http：// $ GATEWAY_URL / productpage`，其中 `$GATEWAY_URL`是 ingress 的外部IP地址，
```
Comment (fleeto) [inline content_zh/docs/tasks/traffic-management/traffic-shifting/index.md]: 似乎有些不可描述的空格？

Comment (fleeto) [reply content_zh/docs/tasks/traffic-management/traffic-shifting/index.md]: `http：// $ GATEWAY_URL / productpage` 这个网址中的空格是不是编辑器加进去的

**Pull request merged**